### ISSUE TITLE
SPM fails to resolve

### ISSUE THREAD

**Issue by philiprenich** [okta/okta-oidc-ios#257]
When trying to install the package via SPM I am receiving this error:

```
Showing All Messages
Source files for target OktaOidc should be located under 'Sources/OktaOidc', or a custom sources path can be set with the 'path' property in Package.swift
```

SPM support was just merged in yesterday (#254) so I'm expecting there's something simple I'm missing or my environment is slightly different. Looking at the repo, the files are definitely under that folder, so I'm not sure what XCode is whining about.

Any idea what's going on here or how to work around it?

**Comment by mikenachbaur-okta** [comment]
SPM support has been merged into `master`, but hasn't been tagged as a release yet. Can you try changing to use the master branch to see if it resolves your issue?

Once we tag the release, the "Latest Version" option should work as expected.
 
![Screen Shot 2020-12-02 at 2 30 42 PM](https://user-images.githubusercontent.com/74688448/100939342-fa199f80-34aa-11eb-911a-e617fbda2466.png)

**Comment by philiprenich** [comment]
Like I said, something simple I'm missing (or being dumb about 😬)
That works just fine, thank you!

**Comment by mikenachbaur-okta** [comment]
@philiprenich I just published the [3.9.3 release](https://github.com/okta/okta-oidc-ios/releases/tag/3.9.3 ). You should be able to safely use the "Version" rule instead of master branch for SPM.
